src/main/java/com/undertheriver/sgsg/foler/domain/Color.java

```diff
@@ -0,0 +1,5 @@
+package com.undertheriver.sgsg.foler.domain;
+
+public enum Color {
```

Comment:
```suggestion
public enum FolderColor {
```
엔티티 내에서만 사용하는 enum이라면 다른곳에서 혼용되지않게 FolderColor도 괜찮겠네요